evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -80,7 +80,8 @@ def validate(self, X, y):
 
         y = infer_feature_types(y)
         is_supported_type = y.ww.logical_type.type_string in numeric_and_boolean_ww + [
-            ww.logical_types.Categorical.type_string
+            ww.logical_types.Categorical.type_string,
+            ww.logical_types.Unknown.type_string,
```

Comment:
Why add `unknown` type as a supported type for `y`? Seems like if it's a classification problem, the type would be guessed as `categorical` if it was string. Are there cases where it could be `unknown`?

Reply:
you're right, I took the wrong tack on that.